gitpod-io/website
# whitepaper component on `/customers` & `/for/enterprise`

## Description

Whitepaper Signup component on `/customers` & `/for/enterprise` page.

## Related Issue(s)



Fixes #3328

## How to test

Open the following pages, scroll down & see :eyes:

- [`/customers`](https://deploy-preview-3330--gitpod-kumquat.netlify.app/customers)
- [`/for/enterprise`](https://deploy-preview-3330--gitpod-kumquat.netlify.app/for/enterprise)



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3330"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
